### DISCUSSION TITLE
ScrollView interfering with buttons

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#9225] [Q&A] [answered]
In skia if the scroll bar goes over a certain length it blocks buttons/other elements from being clicked. Is that a bug or is this intended Skia (WPF specifically) behavior? An example of this is having a Nested RelativePanel below a combo box and as you scroll down the combo box loses the ability to be clicked.

**Comment by francoistanguay** [comment, marked as answer]
Most probably an issue. Would you be kind enough to open an issue with a minimal repro so we can look into it?

**Reply by bstrother1** [reply]
Alright, can do.
